regex-assembly/942300.ra

```diff
@@ -3,24 +3,58 @@
 
 ##!+ i
 
-\)\s*?when\s*?\d+\s*?then
-[\"'`]\s*?#
-[\"'`]\s*?--
-[\"'`]\s*?{
-\/\*!\s?\d+
-\bbinary\s*?\(\s*?\d
-\bchar\s*?\(\s*?\d
-\bchr\s*?\(\s*?\d
-\band\s+\s*?\w+\(
-\bnand\s+\s*?\w+\(
-\bor\s+\s*?\w+\(
-\bxor\s+\s*?\w+\(
-\bxxor\s+\s*?\w+\(
-\bdiv\s+\s*?\w+\(
-\blike\s+\s*?\w+\(
-\bbetween\s+\s*?\w+\(
-\bnot\s+\s*?\w+\(
-\bregexp\s+\s*?\w+\(
-\brlike\s+\s*?\w+\(
-\|\|\s+\s*?\w+\(
-\&\&\s+\s*?\w+\(
+##!> define required-spaces-plus-sql-function-call \s+\w+\(
+##!> define optional-spaces-plus-sql-function-call \s*\w+\(
```

Comment:
I think this should use the same space logic as the others. I see no reason to treat this differently:
```suggestion
##!> define lazy-optional-spaces-plus-sql-function-call \s*?\w+\(
```

Reply:
updated in PR to change all occurrences.